firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/index.ts

```diff
@@ -423,6 +430,7 @@ export class FirestoreBigQueryEventHistoryTracker
       if (this.config.wildcardIds) {
         schema.fields.push(documentPathParams);
       }
+
       const latestSnapshot = latestConsistentSnapshotView(
```

Comment:
The `latest` BigQuery view also needs to be adjusted as it's not possible to group a JSON column.

This is what comes up when you have `data` as a `JSON` field, with the auto-created`collection_raw_latest` view.
<img width="1429" alt="image" src="https://user-images.githubusercontent.com/10195482/194698281-561768c2-e01b-48aa-b3ff-d6333118fea9.png">